e2e/wdio/core/tests/notification.e2e-spec.ts

```diff
@@ -0,0 +1,272 @@
+import { NotificationPo } from '../pages/notification.po';
+import {
+    checkElementScreenshot,
+    click,
+    getElementArrayLength, getText, isElementClickable, isElementDisplayed,
+    refreshPage, saveElementScreenshot,
+    scrollIntoView, waitForElDisplayed
+} from '../../driver/wdio';
+
+import {
+    titleText, descriptionText, metaDataText, metaDataText2, notificationText, templateAsContentApproveText,
+    templateAsContentRejectText, templateAsContentCloseText, componentAsContentCancelText, componentAsContentCloseText,
+    notificationGroupText, thisNotificationGroupText, notificationGroupApproveText, componentAsContentApproveText,
+    notificationGroupCancelText, notificationCloseText, objectApproveText, objectCancelText, objectCloseText
+} from '../fixtures/appData/notification-contents';
+
+describe('Notification test suite', function() {
+    const notificationPage = new NotificationPo();
+    const {
+        notificationButton, buttonMoreInfo, notificationBlock, buttonClose, buttonApprove, buttonCancel, notificationTitle,
+        notificationDescription, notificationMetadata, notificationAvatar, notificationIndicator, notificationSpan,
+        indicatorSuccess, indicatorWarning
+    } = notificationPage;
+
+    beforeAll(() => {
+        notificationPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(notificationButton);
+    }, 1);
+
+    describe('Check orientation', function() {
+        it('Verify LTR / RTL orientation', () => {
+            notificationPage.checkRtlSwitch();
+        });
+    });
+
+    describe('Check Notification example', function() {
+
+        it('verify notification creation and all elements are displayed', () => {
+            scrollIntoView(notificationButton);
+            click(notificationButton);
+            const blockLength = getElementArrayLength(notificationBlock);
+            for (let i = 0; i < blockLength; i++) {
+                scrollIntoView(notificationBlock, i);
+                expect(isElementDisplayed(notificationBlock, i)).toBe(true, `block with index ${i} not displayed`);
+                expect(isElementDisplayed(notificationAvatar, i)).toBe(true, `avatar with index ${i} not displayed`);
+                expect(isElementDisplayed(notificationIndicator, i)).toBe(true, `indicator with index ${i} not displayed`);
+                expect(isElementClickable(buttonMoreInfo, i)).toBe(true, `button more with index ${i} not clickable`);
```

Comment:
'button more with index..' is hard to understand. better: 'more info button with index... '